src/Data/Integer/DivMod.agda

```diff
@@ -98,8 +100,9 @@ a≡a%ℕn+[a/ℕn]*n -[1+ n ] sd@(ℕ.suc d) with (ℕ.suc n) NDM.divMod (ℕ.s
     open ≡-Reasoning
 
     fin-inv : ∀ d (k : Fin d) → + (ℕ.suc d) - + ℕ.suc (Fin.toℕ k) ≡ + (d ℕ.∸ Fin.toℕ k)
-    fin-inv (ℕ.suc n) Fin.zero    = refl
-    fin-inv (ℕ.suc n) (Fin.suc k) = ⊖-≥ {n} {Fin.toℕ k} (NProp.<⇒≤ (FProp.toℕ<n k))
+    fin-inv d k with ℕ.suc d ℕ.<ᵇ ℕ.suc (Fin.toℕ k) | NProp.<ᵇ⇒< (ℕ.suc d) (ℕ.suc (Fin.toℕ k))
+    ... | false | p = refl
+    ... | true  | p = ⊥-elim (NProp.≤⇒≯ (ℕ.s≤s (FProp.toℕ≤n k)) (p _) )
```

Comment:
Isn't this a direct consequence of `m-n≡m⊖n` and `⊖-≥`? That would give us a
cleaner proof using equational reasoning.